<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessWindow</class>
 <widget class="QWidget" name="ProcessWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>621</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Inter</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>611</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>361</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="goToFinishButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>540</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Пропустить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextGenButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>550</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="prevGenButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>550</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;График функции качества&lt;/p&gt;&lt;p&gt;решения:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Номер поколения:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Топ решений:</string>
   </property>
  </widget>
  <widget class="QLabel" name="genNum">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>19</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
